script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_2.txt

```diff
@@ -28,7 +28,7 @@ It's really weird to go into a girl's room and ask her to let you watch her TV.
 Even Kohaku-san is looking at me in disgust――――or maybe not?
 <Page9>
 "Now that you mention it, that's true.
-You've been living in the Arima house until now. The atmosphere in this mansion must have been quite heavy all of a sudden, right?"
+You've been living in the Arima house until now. The heavy atmosphere in this mansion must have been quite abrupt, right?"
```

Comment:
Not sure this is right - I think this is more "The heavy atmosphere in this mansion must have been a shock to the system [after living in the Arima household]"
> 志貴さんは今まで有間のお家で暮らしていたんですもの。いきなりこの屋敷の空気は重かったでしょう？